SteamKit2/SteamKit2/Networking/Steam3/CMServerList.cs

```diff
@@ -0,0 +1,361 @@
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.202" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.202" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.202" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.203" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.203" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.203" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.204" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.204" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.204" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.205" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.205" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.64.200.205" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.9" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.9" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.9" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.10" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.10" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.10" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.11" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.11" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.11" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.12" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.12" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.12" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.13" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.13" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.13" ), 27019 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.14" ), 27017 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.14" ), 27018 ) );
+                Add( new IPEndPoint( IPAddress.Parse( "208.78.164.14" ), 27019 ) );
+            }
+        }
+
+        /// <summary>
+        /// Load a list of servers from the Steam Directory
+        /// </summary>
+        /// <param name="cellid">Cell ID</param>
+        public Task LoadListFromDirectoryAsync( int cellid = 0 )
+        {
+            return LoadListFromDirectoryAsync( cellid, CancellationToken.None );
+        }
+
+        /// <summary>
+        /// Load a list of servers from the Steam Directory
```

Comment:
This should have a better description of what it does, aka it should say that it's going to replace current servers, and should be used once before initial connection.